addons/edi/models/edi_document.py

```diff
@@ -260,6 +261,52 @@ class EdiDocument(models.Model):
 
     fail_fast = fields.Boolean(related="doc_type_id.fail_fast", readonly=True)
 
+    # Processing statistics
+    waiting_time_start = fields.Datetime(string="Waiting start", readonly=True)
```

Comment:
Is it thought to be a one2many with edi.document example edi.document.progress which have fields, 
start,end,total,processed phase(waiting,preparing and executing maybe)

It seems like are similar fields added all to edi.document currently

Reply:
Implemented in d2d0059987663e679e903383986dc3a068e681fa

Reply:
Resolving , but i still believe is better to be in one2many

Reply:
Relevant discussion:
RKV:
```
Another question about SE-1166 (EDI processing times and no. of records, https://unipart.atlassian.net/browse/SE-1166) to the POs (just had a call with Armand where he shared experience on Collingwood):

1. The total records and total records processed — should those be updated realtime (similar to the progress log), while the document is still executing? And here the "total records processed" means currently executed bunch?

2. The timing information are now added to EDI document as in the attached screenshot: this would allow for filtering documents based on these fields (Miquel clarified in the notes that it is required to "run reports on them"); OR is it okay to show these timing details in a list view (similar to say, sale order lines; which is a bit more tricky when filtering edi documents based on these fields)?
```

AP:
```
1. Let's leave the real time updates, just once the doc has been processed is fine.

2. Having the metrics on the form is fine. We can still search and report on them from there.
```